src/convertToObject.js

```diff
@@ -6,7 +6,11 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resut = {};
+
+  const array = sourceString.split(';');
+
+  
 }
 
 module.exports = convertToObject;
```

Comment:
The 'convertToObject' function is not yet complete, and therefore exporting it at this stage is premature. Ensure that the function is fully implemented and tested before exporting it.